web/webpack.dev.js

```diff
@@ -14,7 +14,7 @@ const webpackDev = {
     },
     proxy: {
       '/api': {
-        target: `http://localhost:5000/`,
+        target: `http://localhost:${process.env.MARQUEZ_PORT || 8080}/`,
```

Comment:
I think it might make sense to include the host as well here if we want to go back to this approach.
```javascript
target: `http://${process.env.MARQUEZ_HOST}:${process.env.MARQUEZ_PORT || 8080}}/`,
```

Reply:
You got it 👍